test/Sentry.Tests/Internals/ProcessInfoTests.cs

```diff
@@ -1,10 +1,23 @@
+using System.Runtime.InteropServices;
+
 namespace Sentry.Tests.Internals;
 
 public class ProcessInfoTests
 {
-    [Fact]
+    [SkippableFact]
```

Comment:
What about also:

```suggestion
#if NETCOREAPP
    [SkippableFact]
#endif
```

Reply:
Don't think that should matter all that much really.